screens/LoginScreen.js

```diff
@@ -13,8 +13,8 @@ import {
 import { signUp } from '../redux/actions/user';
 
 class LoginScreen extends Component {
-	componentWillReceiveProps() {
-		if (this.props.currentUser) {
+	componentWillUpdate(newProps) {
+		if (newProps.currentUser) {
```

Comment:
Can you explain to me what you are checking for here? if the user is logged in? Why not just do it in componentDidMount?